src/main/java/org/reactome/addlinks/dataretrieval/pharos/PharosDataRetriever.java

```diff
@@ -49,6 +49,16 @@
 	// curl 'https://ncatsidg-dev.appspot.com/graphql' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: https://ncatsidg-dev.appspot.com' --data-binary '{"query":"query {  targets {    targets(top:100) {      uniprot    }  }}\n"}' --compressed
 	private static int batchSize = 1000;
 
+	public PharosDataRetriever(String retrieverName)
+	{
+		super(retrieverName);
+	}
+
+	public PharosDataRetriever()
+	{
+		super(null);
```

Comment:
Just wanted your input - is it worth passing an empty String instead of null for no retriever name so there are no exceptions due to null values?

Reply:
`CustomLoggable` can handle this, but in the future, I believe that empty strings will work as well.